cmd/podman/machine/init.go

```diff
@@ -100,15 +124,22 @@ func initMachine(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	err = vm.Init(initOpts)
-	if err != nil {
+	finished, err := vm.Init(initOpts)
```

Comment:
Can you elaborate what `finished` means and drop a comment to explain the semantics?

Reply:
Yes good point